setup.py

```diff
@@ -25,7 +25,7 @@ def parse_requirements(filename):
     return [line for line in lineiter if line and not line.startswith("#")]
 
 
-with open("README.md") as readme:
+with open("README.md", encoding="utf8") as readme:
```

Comment:
was the absence causing an error?

Reply:
see https://github.com/capitalone/synthetic-data/issues/280 for error